commit/merkleroot/observation_test.go

```diff
@@ -21,12 +21,13 @@ import (
 	"github.com/smartcontractkit/chainlink-ccip/internal/mocks"
 	"github.com/smartcontractkit/chainlink-ccip/mocks/commit/merkleroot"
 	common_mock "github.com/smartcontractkit/chainlink-ccip/mocks/internal_/plugincommon"
-	reader_mock "github.com/smartcontractkit/chainlink-ccip/mocks/internal_/reader"
+	reader_mock "github.com/smartcontractkit/chainlink-ccip/mocks/pkg/reader"
 
 	"github.com/smartcontractkit/chainlink-ccip/plugintypes"
 )
 
 func Test_Observation(t *testing.T) {
+	t.Skip()
```

Comment:
Shouldnt skip?

Reply:
Oops, I was testing something unrelated to this PR